docker-compose.yml

```diff
@@ -40,6 +40,9 @@ services:
       IDP_CLIENT_ID: "foo"
       DELEGATED_LOGIN_URL: "http://localhost:8082/modialogin/api/start"
       AUTH_TOKEN_RESOLVER: "modia_ID_token"
+      # Just for testing, recommend omitting "CSP_REPORT_ONLY" which will default the value to "false"
+      CSP_REPORT_ONLY: "true"
```

Comment:
vil denne overskrive eller bli overskriven av start-nginx.sh som setter det til false? Eller er det ein logikk her eg ikkje forstår?

Reply:
`docker-compose.yml` er kun i bruk for testene og når man vil kjøre det opp lokalt. Så her setter jeg den til `true` bare for å kunne sjekke i testen at vi bryr oss om `CSP_REPORT_ONLY`.

For apper som tar ibruk docker-imaget, så vil de fleste ikke sette den variablen og da slår defaulten i `start-nginx` inn.